EnvChanger.podspec

```diff
@@ -15,7 +15,7 @@ recompile.
   s.authors          = { 'Gavril Tonev' => 'gavril.tonev@upnetix.com', 'Teodor Marinov' => 'teodor.marinov@upnetix.com' }
   s.source           = { :git => 'https://github.com/scalefocus/EnvChanger.git', :tag => s.version.to_s }
 
-  s.ios.deployment_target = '8.0'
+  s.ios.deployment_target = '9.0'
```

Comment:
You can safely raise the version to 13 I believe.

Reply:
Done.